src/sentry/feedback/endpoints/base.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework.request import Request
+
+from sentry import features
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.feedback.models import Feedback
+
+
+class ProjectFeedbackEndpoint(ProjectEndpoint):
+    def convert_args(self, request: Request, feedback_id, *args, **kwargs):
+        args, kwargs = super().convert_args(request, *args, **kwargs)
+        project = kwargs["project"]
+
+        if request.method != "DELETE" and not features.has(
+            "organizations:user-feedback-ingest", project.organization, actor=request.user
+        ):
+            raise ResourceDoesNotExist
+
+        try:
+            kwargs["feedback"] = Feedback.objects.get(feedback_id=feedback_id)
```

Comment:
Shouldn't this be scoped by project/org? It looks like there would be cross org access here.

Reply:
good call. missed this while copy pastating. will add project and a test to confirm.

Reply:
actually -- since this is inherited from ProjectEndpoint and we call super().convert_args, we validate there that user has access to project/org resources. I'll add project_id in query here just for safety.